db/migrate/20200420143824_move_electronic_pharmacy_stock_rows_into_files.rb

```diff
@@ -0,0 +1,18 @@
+class MoveElectronicPharmacyStockRowsIntoFiles < ActiveRecord::Migration[5.2]
+  class ElectronicPharmacyStockRecord < ApplicationRecord
+    has_one_attached :rows_file
+  end
+
+  def up
+    ElectronicPharmacyStockRecord.all.each do |e|
+      # Don't create files unless there is content (!nil or !{})
+      if e.rows && !e.rows.empty?
```

Comment:
I think you can use present? here
```suggestion
      if e.present?
```